### PR TITLE
Issue 47

### PR DESCRIPTION
**Pull request by JoshBudzynski**
Solves Issue 47

Adds a next and previous button before the arbitrary bone sets we have so far, and displays the information.

Might break when there is a larger data set.

### CODE REVIEW

File: Data_Directory/Bone_Directory/boneset.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            }
+            return response.json();
+        }
+
+        // Function to display the boneset at the current index
+        function displayBoneSet(index) {
+            const boneSetList = document.getElementById('bone-set-list');
+            boneSetList.innerHTML = '';  // Clear previous data
+
+            const set = bonesetData[index];
```
Comment (tluke900) [inline Data_Directory/Bone_Directory/boneset.html:68]: We did this a little differently, working on standardization later would be helpful.

**Review by tluke900 — COMMENTED** [review]
Overall very similar to mine, completed very similarly.

**Comment by kate-holdener** [comment]
It seems like this code does not match the directory structure in main.

File: .DS_Store
Comment (kate-holdener) [inline .DS_Store:1]: Do not add these files to the repo

File: Button.html
Comment (kate-holdener) [inline Button.html:1]: This file is irrelevant to the issue you are working on.

**Review by jacksayshi — CHANGES_REQUESTED** [review]
Thanks for working on this. 
But I think you might have created a new branch based on your old branch. Please restructure your branch based on latest commits in main (As we have agreed on a format 
for now). And make changes to html in that file structure.

**Pull request closed without merging**